frontend/src/pages/EnvironmentSettings/Components/AuditParameterTable/AddAuditParameterRow/AddAuditParameterRow.tsx

```diff
@@ -1,9 +1,13 @@
 import Select from 'components/Select/Select';
-import { Add } from 'icons';
+import Add from 'icons/Add';
```

Comment:
Can these be imported as `AddIcon`, `CloseIcon` and `DoneIcon`? I would make it clear that they are merely icons, not full-fledged components.

You can do `import Add as AddIcon from 'icons/Add'`